packages/core/nuxt-theme-module/theme/pages/Product.vue

```diff
@@ -234,6 +236,16 @@ export default {
       });
     };
 
+    const addToCart = async (product, quantity) => {
+      await addItem({ product, quantity });
+      sendNotification.cart.addItem({
+        productName: product._name,
+        error: error.value.addItem ? error.value.addItem.message : false,
+        onClick: () => $router.push('/checkout/personal-details'),
+        $i18n
```

Comment:
```suggestion
        $i18n
```

you can get this through `useContext()` inside `sendNotification`

Reply:
I think `useContext()` must be called within a setup function so we cannot use it inside `sendNotification `

Reply:
I think we can if addItem is always executed in the `setup` like here